circuits/src/main.nr

```diff
@@ -0,0 +1,33 @@
+use dep::std;
+use dep::ecrecover;
+
+global CANDIDATES: Field = 2;
+
+fn main(
+  pub_key: [u8; 64],
+  signature: [u8; 64],
+  hashed_message: pub [u8; 32],
+  nullifier: pub [u8; 32],
+) {
+  let (pub_key_x, pub_key_y) = split_uncompressed_pub_key(pub_key);
+  
+  let address: Field = ecrecover::ecrecover(pub_key_x, pub_key_y, signature, hashed_message);
```

Comment:
i think we can do it this way to avoid the splitting function below:

```suggestion
    let key = ecrecover::secp256k1::PubKey::from_unified(pub_key);
    let address = key.ecrecover(signature, hashed_message);
```